scripts/stubsabot.py

```diff
@@ -323,12 +323,15 @@ def _plural_s(num: int, /) -> str:
     return "s" if num != 1 else ""
 
 
-@dataclass
+@dataclass(repr=False)
 class DiffAnalysis:
     MAXIMUM_NUMBER_OF_FILES_TO_LIST: ClassVar[int] = 7
     py_files: list[FileInfo]
     py_files_stubbed_in_typeshed: list[FileInfo]
 
+    def __repr__(self) -> str:
+        return f"<DiffAnalysis object at {id(self)}>"
```

Comment:
Isn't this just the default? I don't think you need both this override and setting `repr=False` in the decorator.

Reply:
oh, good point